content/adr/adopt-semantic-versioning.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Adopt semantic versioning for releases"
+date: "2022-04-13"
```

Comment:
Let's fix the date.

Reply:
@hussainweb Actually this is the date I intentionally gave. Also gave context in [description](https://github.com/axelerant/engg-handbook/pull/235#issue-2808675503)